src/InfoViz/Native/ParallelCoordinates/index.js

```diff
@@ -294,6 +297,7 @@ function parallelCoordinate(publicAPI, model) {
           .attr('width', glyphSize)
           .attr('height', glyphSize)
           .style('color', d.legend.color) // Firefox SVG use color bug workaround fix
+          .classed(style.clickable, (data, idx) => d.annotated)
```

Comment:
It's doing what I want, just confusing to readers.  I'll get rid of `(data, idx)`, as what I want is `d.annotated`.